lib/ancestry/has_ancestry.rb

```diff
@@ -31,7 +36,7 @@ def has_ancestry options = {}
     self.touch_ancestors = options[:touch] || false
 
     # Validate format of ancestry column value
-    validates_format_of ancestry_column, :with => Ancestry::ANCESTRY_PATTERN, :allow_nil => true
+    validates_format_of ancestry_column, :with => ancestry_pattern, :allow_nil => true
```

Comment:
There are a few examples in the [tests](https://github.com/stefankroes/ancestry/pull/262/files#diff-55b6e672c474d6ae1a8037ac00471dbfR37).
The thought is that the delimiter would not change after the column is configured.